styles/layout.less

```diff
@@ -694,19 +738,22 @@ table.snapshottable {
     }
 }
 
+@access-yes: @text-ok;
```

Comment:
I'm not certain we want to tie the access colors to the text OK/error/warning colors. Consider the case where a theme changes warning to orange -- I don't think an orange info icon is the right message for "eligible".

Reply:
Fair comment -- and I felt uncomfortable using the error/warning colors there. Not sure why I left that, actually. I can either put the colors in directly, or define additional color names--do you have a preference?

Reply:
Actually, never mind ... because that was already a level of indirection. I'll just assign the colors to those variables directly.